src/main/java/Main.java

```diff
@@ -1,73 +1,103 @@
         init();
 
+        //noinspection InfiniteLoopStatement
         while (true) {
-            System.out.println("******** Register *********");
-            System.out.println("please do choice");
-            System.out.println("1.Login 2.Register 3.exit 4.showMe list");
             int choice = Integer.parseInt(scanner.nextLine());
 
             switch (choice) {
-                case CommandConstants.LOGIN_CODE:
+                case CommandConstants.LOGIN_CODE -> {
                     System.out.println("Your username");
                     String inputUsername = scanner.nextLine();
                     System.out.println("password");
                     String inputPassword = scanner.nextLine();
                     LoginCommand loginCommand = new LoginCommand();
                     loginCommand.execute(inputUsername, inputPassword);
-                    break;
-
-                case CommandConstants.REGISTER_CODE:
+                    menuPrinter.print(UserSession.currentUser.getRole());
+                }
+                case CommandConstants.REGISTER_CODE -> {
                     System.out.println("Create your account");
                     System.out.println("Your userName");
-                    String createUserName = scanner.nextLine();
-                    if (UserContainer.getUsers().toString().contains(createUserName)) {
-                        System.out.println("such a user exist already");
-                        return;
-                    }
+                    String inputUsername = scanner.nextLine();
                     System.out.println("create password ");
-                    String createPassword = scanner.nextLine();
+                    String inputPassword = scanner.nextLine();
                     System.out.println("\ninput your name");
-                    String createName = scanner.nextLine();
+                    String inputName = scanner.nextLine();
                     System.out.println("\ninput your surname");
-                    String createSurname = scanner.nextLine();
+                    String inputSurname = scanner.nextLine();
                     System.out.println("your role");
                     String inputRole = scanner.nextLine();
-                    UserRole createRole = UserRole.valueOf(inputRole.toUpperCase());
-                    new RegisterCommand().register(createUserName, createPassword, createName, createSurname, createRole);
-                    break;
-
-                case CommandConstants.EXIT_CODE:
+                    UserRole convertToUserRole = UserRole.valueOf(inputRole.toUpperCase());
+                    User user = new User(
+                            inputUsername,
+                            inputPassword,
+                            inputName,
+                            inputSurname,
+                            convertToUserRole,
+                            UserStatus.ACTIVE);
```

Comment:
you have fields:
- **input**Username
- **input**Name
- **input**Surname
and **convertTo**UserRole

one style everywhere